docs/sources/tempo/api_docs/_index.md

```diff
@@ -53,6 +53,7 @@ For externally supported gRPC API, [refer to Tempo gRPC API](#tempo-grpc-api).
 | [Compactor ring status](#compactor-ring-status) | Compactor |  HTTP | `GET /compactor/ring` |
 | [Status](#status) | Status |  HTTP | `GET /status` |
 | [List build information](#list-build-information) | Status |  HTTP | `GET /api/status/buildinfo` |
+| [MCP Server](https://grafana.com/docs/tempo/<TEMPO_VERSION>/api_docs/mcp-server) (*) | MCP |   | `/api/mcp` |
```

Comment:
Q: should we have a section in docs page for MCP endpoint like we do for others? or it doesn't make sense?